### DISCUSSION TITLE
DS 3.0版本伪集群模式下，使用本地资源配置，无法上传、创建文件。

### DISCUSSION THREAD

**Discussion by DF1024** [apache/dolphinscheduler#11853] [Q&A] [answered]
按照官网的本地资源配置了api-server/conf/common.properties 和 worker-server/conf/common.properties。
![ade22f8d9e1cc5a89b89f5a0f61e81b](https://user-images.githubusercontent.com/44440466/189057018-019a13f9-03ae-4f59-b145-8f0d011ae841.jpg)

官网上说无需部署Hadoop，但是实际上传时候，报错信息却是用到了Hadoop。
![42bfbc38e91396482efd1521139b9d2](https://user-images.githubusercontent.com/44440466/189057147-69a11caa-3912-448f-9021-e0179ead8911.jpg)

对/tmp/dolphinscheduler 目录是有读写权限的。
![bec30a671bd75da085145e8b09b1690](https://user-images.githubusercontent.com/44440466/189057273-bdd7b407-c988-40ba-8789-b62fdc3ec0f1.jpg)
-----------------------
请问一下这个是什么问题导致的呢

**Comment by DarkAssassinator** [comment, marked as answer]
Just support HDSF and S3, not support local.

**Reply by DF1024** [reply]
I see . The standalone supports local but I used the pseudo-cluster

**Comment by F-PHantam** [comment]
update the resource.upload.path to the value where you have read and write right. having a try!
